### PR TITLE
863264 Upgraded the samples from .Net 6.0 to .Net 8.0

### PR DESCRIPTION
**Pull request by VijayadharshiniMathiyalagan**
Description:
Upgraded the following samples

1. Choose files from list
2. Custom tool bar
3. Open local file
4. Open password protected file
5. Open URL file
6. Page navigation
7. Property synchronisation
8. Busy indicator style
9. zoom

### CODE REVIEW

File: Choose PDF From List/ChoosePDFFromList.csproj [hunk 1]
```diff
@@ -65,7 +65,19 @@
 	</ItemGroup>
 
 	<ItemGroup>
-	  <PackageReference Include="Syncfusion.Maui.PdfViewer" Version="21.1.37" />
+		<PackageReference Include="Microsoft.Maui.Controls" Version="$(MauiVersion)" />
+		<PackageReference Include="Microsoft.Maui.Controls.Compatibility" Version="$(MauiVersion)" />
+		<PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="8.0.0" />
+		<PackageReference Include="Syncfusion.Compression.NET" Version="*" />
```
Comment (Deepak1799) [inline Choose PDF From List/ChoosePDFFromList.csproj:71]: It is not required to include all the syncfusion PDF Viewer packages when you refer from **Nuget org**. Please check and optimize

**Review by Deepak1799 — CHANGES_REQUESTED** [review]
Requested changes

File: Choose PDF From List/ChoosePDFFromList.csproj (hunk 1 above)
Comment (VijayadharshiniMathiyalagan) [reply Choose PDF From List/ChoosePDFFromList.csproj]: Optimised

File: Zoom/Zoom.csproj [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
```
Comment (Deepak1799) [inline Zoom/Zoom.csproj:3]: image

NuGet versions was not changed for some samples. Did you run and check?

**Review by Deepak1799 — CHANGES_REQUESTED** [review]
Requested changes

File: Zoom/Zoom.csproj (hunk 2 above)
Comment (VijayadharshiniMathiyalagan) [reply Zoom/Zoom.csproj:3]: yes sir run it. it was working

**Review by Deepak1799 — APPROVED** [review]
Approved the changes

**Pull request merged**